indexer/road_shields_parser.cpp

```diff
@@ -565,6 +568,23 @@ class CyprusRoadShieldParser : public SimpleRoadShieldParser
   }
 };
 
+class TurkeyRoadShieldParser : public SimpleRoadShieldParser
+{
+public:
+  explicit TurkeyRoadShieldParser(std::string const & baseRoadNumber)
+    : SimpleRoadShieldParser(baseRoadNumber, {
+                                              {"O-", RoadShieldType::Generic_Orange}, // Black font. Hexagon.
+                                              {"D.", RoadShieldType::Generic_Blue}, // White font.
+                                              {"D-", RoadShieldType::Generic_Blue}, // White font.
+                                              {"D", RoadShieldType::Generic_Blue}, // White font.
+                                              {"D ", RoadShieldType::Generic_Blue}}) // White font.
+  {
+  }
+    : NumericRoadShieldParser(baseRoadNumber, {{01, 81, RoadShieldType::Generic_White} }}) //Blue font.                                             
```

Comment:
It won't compile like this.
Remove (comment) "NumericRoadShieldParser" entry and leave todo. So we can merge your PR now, and I can think about how to implement it properly later.